core/arch/x86/decode_table.c

```diff
@@ -3914,8 +4435,13 @@ const instr_info_t prefix_extensions[][8] = {
     {INVALID,   0xf20f7132, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
     {INVALID,     0x0f7132, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
     {INVALID,   0xf30f7132, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {OP_vpsrlw,  0x660f7132, "vpsrlw", Hx, xx, Ib, Ux, xx, mrm|vex, x, END_LIST},
+    {OP_vpsrlw, 0x660f7132, "vpsrlw", Hx, xx, Ib, Ux, xx, mrm|vex, x, END_LIST},
```

Comment:
Repeating: I would avoid any of these whitespace changes: it makes this diff confusing.  Its size already makes it hard to look at all the changes and having superfluous changes does not help readability.

Reply:
Reverted these kind of changes. Done.